lib/events/usageevents/usageevents.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package usageevents
+
+import (
+	"context"
+
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/api/types"
+	apievents "github.com/gravitational/teleport/api/types/events"
+	"github.com/gravitational/teleport/lib/events"
+	"github.com/gravitational/teleport/lib/services"
+)
+
+// UsageLogger is a trivial audit log sink that forwards an anonymized subset of
+// audit log events to Teleport.
+type UsageLogger struct {
+	// Entry is a log entry
+	*logrus.Entry
+
+	// inner is a wrapped audit log implementation
+	inner events.IAuditLog
```

Comment:
Do you need a full `IAuditLog`, or is an emitter enough?
```suggestion
	inner events.Emitter
```